ansible/pbTestScripts/qemu_test_script.sh

```diff
@@ -139,24 +133,27 @@ done
 		"S390X" )
 			export MACHINE="s390-ccw-virtio";
```

Comment:
Not sure why we have `;` at the end of each line here - you only need `;;` at the end of the block

Reply:
I was probably missing `Java` or `C` that day ... Fixed now